cachito/web/config.py

```diff
@@ -7,6 +7,11 @@
 
 class Config(object):
     """The base Cachito Flask configuration."""
+    # Additional loggers to set to the level defined in CACHITO_LOG_LEVEL
+    CACHITO_ADDITIONAL_LOGGERS = []
+    # This sets the level of the "flask.app" logger, which is accessed from current_app.logger
+    CACHITO_LOG_LEVEL = 'INFO'
```

Comment:
Can we set this to `DEBUG` for `DevelopmentConfig` and `TestingConfig` ?

Reply:
Good point! I just added that.